docs/tutorials/old_comp_tutorials/tut-add-connector-security.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Add an Identity Provider to Dex
```

Comment:
not sure if Dex is still relevant?

Reply:
According to [PB's comment](https://github.com/kyma-project/kyma/issues/10876#issuecomment-800365786), it should probably be removed 🤔